src/GraphQL/Conversion/ValueConverter.cs

```diff
@@ -114,6 +114,11 @@ static ValueConverter()
             Register(typeof(double), typeof(decimal), value => Convert.ToDecimal(value, NumberFormatInfo.InvariantInfo));
 
             Register(typeof(string), typeof(Uri), value => value is string s ? new Uri(s) : (Uri)value);
+            // registering such a default conversion for string->byte[] seems useful
+            Register(typeof(string), typeof(byte[]), value => Convert.FromBase64String((string)value));
```

Comment:
@Shane32 This is worth noting in the documentation.

Reply:
Probably should be noted in `schema-types.md` as well as in the upgrade notes; would you agree?

Reply:
yes

Reply:
There will be another change with value conversion soon.